sensing/pointcloud_preprocessor/launch/blockage_diag.launch.xml

```diff
@@ -5,7 +5,8 @@
   <arg name="angle_range" default="[0.0, 360.0]"/>
   <arg name="vertical_bins" default="40"/>
   <arg name="is_channel_order_top2down" default="true"/>
```

Comment:
You can remove these after those parameters are in the schema file and parameters file
"horizontal_ring_id"
"vertical_bins"
"angle_range"
"is_channel_order_top2down"

Reply:
Oh! Didn't realized, nice suggestion! Changed.